modules/aaxBlockmeterRtdProvider.js

```diff
@@ -0,0 +1,45 @@
+import {isEmptyStr, isStr, logError, isFn} from '../src/utils.js';
+import {submodule} from '../src/hook.js';
+import { loadExternalScript } from '../src/adloader.js';
+
+const MODULE = 'aaxBlockmeter';
+const ADSERVER_TARGETING_KEY = 'atk';
+const BLOCKMETER_URL = 'https://c.aaxads.com/aax.js';
```

Comment:
Please make this a configuration param. If you fall back to it when missing, please lock it to a version. We want publishers using your module to be extremely aware there is js loaded that the project hasn't warranted or reviewed. Please also add an external js disclosure to your docs PR.

Reply:
Hi, @patmmccann, have done the changes please r[eview the changes](https://github.com/prebid/Prebid.js/pull/9135/commits/c425d7af0e8c43a1d9d52ecd873f4b1ddf39af2a) and [docs](https://github.com/prebid/prebid.github.io/pull/4095/commits/5d1b65fec6b723bc963cca038628691bad1009e6)